test/populate_test/bdl_commands_test/circulation_loop_test.py

```diff
@@ -1,29 +1,972 @@
+        self.system = System("System 1", self.rmd)
+        self.zone = Zone("Zone 1", self.system, self.rmd)
+        self.chiller = Chiller("Chiller 1", self.rmd)
+
+        # Create Chilled Water/Hot Water Temperature Reset Schedules
+        self.temp_reset_day_schedule = DaySchedulePD(
+            "Temp Reset Day Schedule", self.rmd
+        )
+        self.temp_reset_week_schedule = WeekSchedulePD(
+            "Temp Reset Week Schedule", self.rmd
+        )
+        self.temp_reset_annual_schedule = Schedule(
+            "Temp Reset Annual Schedule", self.rmd
+        )
+
+        # Create Loop Operation Schedules
+        self.loop_operation_day_schedule = DaySchedulePD(
+            "Loop Operation Day Schedule", self.rmd
+        )
+        self.loop_operation_week_schedule = WeekSchedulePD(
+            "Loop Operation Week Schedule", self.rmd
+        )
+        self.loop_operation_annual_schedule = Schedule(
+            "Loop Operation Annual Schedule", self.rmd
+        )
+
+        # Create System Fan Schedules
+        self.fan_schedule_day_schedule = DaySchedulePD("Fan Day Schedule", self.rmd)
+        self.fan_schedule_week_schedule = WeekSchedulePD("Fan Week Schedule", self.rmd)
+        self.fan_schedule_annual_schedule = Schedule("Fan Annual Schedule", self.rmd)
+
+        # Generate annual calendar for schedules
+        generated_calendar = schedule_funcs.generate_year_calendar(2020, "WEDNESDAY")
+        self.temp_reset_annual_schedule.annual_calendar = generated_calendar
+        self.loop_operation_annual_schedule.annual_calendar = generated_calendar
+        self.fan_schedule_annual_schedule.annual_calendar = generated_calendar
+
+        # Populate default keyword value pairs for all tests
+        self.pump.keyword_value_pairs = {BDL_PumpKeywords.NUMBER: "1"}
+        self.temp_reset_day_schedule.keyword_value_pairs = {
+            BDL_DayScheduleKeywords.TYPE: BDL_ScheduleTypes.RESET_TEMP,
+            BDL_DayScheduleKeywords.OUTSIDE_HI: "100",
+            BDL_DayScheduleKeywords.OUTSIDE_LO: "50",
+            BDL_DayScheduleKeywords.SUPPLY_HI: "95",
+            BDL_DayScheduleKeywords.SUPPLY_LO: "45",
+        }
+        self.temp_reset_week_schedule.keyword_value_pairs = {
+            BDL_WeekScheduleKeywords.TYPE: BDL_ScheduleTypes.RESET_TEMP,
+            BDL_WeekScheduleKeywords.DAY_SCHEDULES: [
+                "Temp Reset Day Schedule" for i in range(12)
+            ],
+        }
+        self.temp_reset_annual_schedule.keyword_value_pairs = {
+            BDL_ScheduleKeywords.TYPE: BDL_ScheduleTypes.RESET_TEMP,
+            BDL_ScheduleKeywords.WEEK_SCHEDULES: "Temp Reset Week Schedule",
+            BDL_ScheduleKeywords.MONTH: "12",
+            BDL_ScheduleKeywords.DAY: "31",
+        }
+        self.loop_operation_day_schedule.keyword_value_pairs = {
+            BDL_DayScheduleKeywords.TYPE: BDL_ScheduleTypes.ON_OFF,
+            BDL_DayScheduleKeywords.VALUES: ["1" for i in range(24)],
```

Comment:
I am making edits to this branch so no need to address, just FYI -
`i` is not being used for anything so `["1" for i in range(24)]` can be simplified to `["1"]*24`